src/com/facebook/buck/apple/project_generator/ProjectGenerator.java

```diff
@@ -1300,6 +1301,18 @@ private PBXNativeTarget generateBinaryTarget(
           appendConfigsBuilder.put("SWIFT_INCLUDE_PATHS", "$BUILT_PRODUCTS_DIR");
         }
 
+        ImmutableList.Builder<String> targetSpecificSwiftFlags = ImmutableList.builder();
+        if (targetNode.getConstructorArg() instanceof SwiftCommonArg) {
+          targetSpecificSwiftFlags.addAll(
+              convertStringWithMacros(
+                  targetNode,
+                  ((SwiftCommonArg) targetNode.getConstructorArg()).getSwiftCompilerFlags()));
+        }
+
+        Iterable<String> otherSwiftFlags =
+            Iterables.concat(
+                swiftBuckConfig.getFlags("compiler_flags").orElse(DEFAULT_SWIFTFLAGS),
```

Comment:
Let's use `swiftBuckConfig.getFlags()` instead because that's what's used by `SwiftCompile`, so that we pass the same sets of flags in Xcode + `buck build`.

Reply:
I did it like this to keep it consistent with the rest of the file, which all has getFlags with a parameter. With a parameter seems better than just `getFlags`, as that is a bit generic. If you have `swiftBuckConfig.getFlags()` which flags are you getting? Now its compiler flags, but that seems arbitrary.

Reply:
My main concern here is the hardcoded string value which can get out of sync with other places and end up in different flags being passed in Xcode vs buck build. Can we instead expose enum in `SwiftBuckconfig`that defines all valid flags?